service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/GatekeepingOrderEventHandlerTest.java

```diff
@@ -32,15 +33,13 @@
 import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.CTSC_INBOX;
 import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_CODE;
 import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.LOCAL_AUTHORITY_EMAIL_ADDRESS;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
 
 @ExtendWith(MockitoExtension.class)
 class GatekeepingOrderEventHandlerTest {
     private static final Long CASE_ID = 12345L;
     private static final SDONotifyData NOTIFY_DATA = mock(SDONotifyData.class);
-    private static final DocumentReference ORDER = mock(DocumentReference.class);
-    private static final GatekeepingOrderEvent.GatekeepingOrderEventBuilder BASE_EVENT = GatekeepingOrderEvent.builder()
```

Comment:
problem is static field that is mutated between tests. Because it is static, change is kept and present in next test invocation

Reply:
🤦 , thanks for fixing